server/chat-plugins/helptickets.ts

```diff
@@ -1343,16 +1380,14 @@ export const commands: ChatCommands = {
 			// This is a utility only to be used if something goes wrong
 			this.checkCan('makeroom');
 			if (!target) return this.parse(`/help helpticket delete`);
-			const ticket = tickets[toID(target)];
-			if (!ticket) return this.errorReply(this.tr`${target} does not have a ticket.`);
-			const targetRoom = Rooms.get(`help-${ticket.userid}`);
+			const targetRoom = Rooms.get(`help-${toID(target)}`);
 			if (targetRoom) {
 				targetRoom.getGame(HelpTicket)!.deleteTicket(user);
```

Comment:
Is it possible for the room not to have a ticket?

Reply:
I don't believe so?

Reply:
I guess we'll find out.